net/openconnect/README

```diff
@@ -18,6 +18,9 @@ config interface 'MYVPN'
 	# Reconnect after a temporary network down time (in seconds)
 	#option reconnect_timeout '30'
 
+	# Shell command line for using a vpnc-compatible config script (default: "/lib/netifd/vpnc-script")
```

Comment:
Is this script called even without specifying a `--script` parameter? The code below doesn't set any default value.

Reply:
Yes, it's set in the Makefile
https://github.com/openwrt/packages/blob/7e272a577e4f5d3cd736c4d228e18d7e4ed5cdfd/net/openconnect/Makefile#L61